src/bench/bench.cpp

```diff
@@ -43,15 +44,38 @@ void GenerateTemplateResults(const std::vector<ankerl::nanobench::Result>& bench
 
 namespace benchmark {
 
+// map a label to one or multiple priority levels
+std::map<std::string, uint8_t> map_label_priority = {
+    std::make_pair("high", PriorityLevel::HIGH),
+    std::make_pair("medium", PriorityLevel::MEDIUM),
+    std::make_pair("low", PriorityLevel::LOW),
+    std::make_pair("all", 0xff)
```

Comment:
stylistic nit (feel free to ignore):
```suggestion
    {"high", PriorityLevel::HIGH},
    {"medium", PriorityLevel::MEDIUM},
    {"low", PriorityLevel::LOW},
    {"all", 0xff}
```